### ISSUE TITLE
Outdated documentation on Azure deployment

### ISSUE THREAD

**Issue by robintw** [developmentseed/titiler#787]
#### Problem description
The Azure deployment documentation at https://developmentseed.org/titiler/deployment/azure/ seems to be out-dated. The incorrect part seems to be in the Titiler parts rather than the Azure stuff - interestingly.

The following import fails:

```
from titiler.application.routers import cog, mosaic, stac, tms
```

I will try and submit a PR if I can find time, but I'm not quite sure what the modern way of importing those routers is - do I need to import the factories and register the routers from those?

**Comment by robintw** [comment]
Ah, just seen a PR fixing this at https://github.com/developmentseed/titiler/pull/680. Can that be merged soon to stop anyone else running into the same problems that I ran into?

**Comment by vincentsarago** [comment]
@robintw I'll have a look later this. 

I didn't merge before because CI was failing and I was expecting the author to fix the issue

**Comment by vincentsarago** [comment]
merged #680 so I guess we can close here
